python_scripts/01_tabular_data_exploration.py

```diff
@@ -99,11 +101,22 @@
 
 # %%
 numerical_columns = [
-    "age", "education-num", "capital-gain", "capital-loss",
-    "hours-per-week"]
+    "age",
+    "education-num",
+    "capital-gain",
+    "capital-loss",
+    "hours-per-week",
+]
 categorical_columns = [
-    "workclass", "education", "marital-status", "occupation",
-    "relationship", "race", "sex", "native-country"]
+    "workclass",
+    "education",
+    "marital-status",
+    "occupation",
+    "relationship",
+    "race",
+    "sex",
+    "native-country",
```

Comment:
I personnally am not super excited about those cosmetic changes.

First: this formatting makes sense maybe to reduce merge conflicts, but it also maximizes vertical scrolling. So for the reader it is not optimal, in particular for the reader that is not a trained software engineer used to this formatting.

Second, if we all start changing these details, we'll be spending a lot of time on them (possibly with back and forth according to people's preference).